src/pickers/HealthFacilityReferPicker.js

```diff
@@ -0,0 +1,73 @@
+import { healthFacilityLabel, LOCATION_SUMMARY_PROJECTION } from "../utils";
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { useModulesManager, useTranslations, Autocomplete, useGraphqlQuery } from "@openimis/fe-core";
+import _debounce from "lodash/debounce";
+
+const HealthFacilityReferPicker = (props) => {
+  const {
+    onChange,
+    readOnly,
+    required,
+    withLabel = true,
+    withPlaceholder,
+    value,
+    label,
+    filterOptions,
+    filterSelectedOptions,
+    placeholder,
+    multiple,
+    region,
+    district,
+    level,
+  } = props;
+
+  const modulesManager = useModulesManager();
+  const { formatMessage } = useTranslations("location", modulesManager);
+  const [searchString, setSearchString] = useState("");
+  const { data, isLoading, error } = useGraphqlQuery(
+    `
+    query HealthFacilityPicker ($str: String, $region: String, $district: [String], $level: String) {
```

Comment:
Since you're not providing any location details to the query, it should be properly adjusted.